src/DlibDotNet.Native/dlib/array2d/array2d.h

```diff
@@ -17,6 +17,9 @@ ret = new dlib::array2d<__TYPE__>();
 #define array2d_new1_template(__TYPE__, error, type, ...) \
 ret = new dlib::array2d<__TYPE__>(rows, cols);
 
+#define array2d_data_raw_template(__TYPE__, error, type, ...) \
+ret = &(((array2d<__TYPE__>*)array)->[0][0]);
```

Comment:
````
        iterator begin() 
        {
            return data;
        }
````

array_2d  exposes begin method.
You can use it if you want to only access raw data.
What do you think about?

Reply:
Thank you, I've updated the code to use begin() instead of returning the address at `[0][0]``.

Reply:
;)